src/main/java/me/lucien/minesweeper/web/GameController.java

```diff
@@ -29,6 +28,15 @@ public String createRoom(@RequestParam("width") int width, @RequestParam("height
         Room room = new Room(width, height);
         roomMap.put(room.getId(), room);
 
+        Timer timer = new Timer();
+        timer.schedule(new TimerTask() {
+            @Override
+            public void run() {
+                roomMap.remove(room.getId());
+            }
+        }, 60 * 60 * 1000);
```

Comment:
Avoid using constants as much as possible. If you have to, always declare a comprehensive constant first. Or we can make it configurable with application.yml.